feature/account/server/settings/src/main/kotlin/app/k9mail/feature/account/server/settings/ui/incoming/content/IncomingFormItems.kt

```diff
@@ -32,7 +33,11 @@ internal fun LazyListScope.incomingFormItems(
     resources: Resources,
 ) {
     item {
-        Spacer(modifier = Modifier.requiredHeight(MainTheme.sizes.smaller))
+        Spacer(
+            modifier = Modifier
+                .requiredHeight(MainTheme.sizes.smaller)
+                .clearAndSetSemantics { },
```

Comment:
Is this really necessary? It feels odd that a warning would be generated for a `Spacer` with a missing(?) content description. If here, why not in the other places where a `Spacer` is used?

Reply:
I'm not completely sure, I might have targeted the wrong element. Here is the screenshot from the pre-launch report. It looks a bit like the server name field, but that one provides a text description?

![image](https://github.com/user-attachments/assets/77be796d-2452-4056-aa2d-c4f1f38d9b18)